pylds/tools.py

```diff
@@ -74,12 +74,23 @@ def draw_lagrangian_descriptor(LD, LD_type, grid_parameters, tau, p_value, norm
         string_title = ''
         print('Incorrect "LD_type". Valid options: forward, backward, total. Plot will appear without title')
     
-    ax.set_title(string_title, fontsize=12)
-    ax.set_xlabel('$x$', fontsize=18)
-    ax.set_ylabel('$y$', fontsize=18)
+    fig.suptitle(string_title, fontsize=14, y=1.04)
+    ax0.set_title('LD values')
+    ax0.set_xlabel('$x$')
+    ax0.set_ylabel('$y$')
+    
+    fig.colorbar(con0, ax=ax0, ticks=np.linspace(np.nanmin(LD),np.nanmax(LD),11),format='%.2f')
+    
+    gx,gy = np.gradient(LD, 0.05, 0.05)
+    scalar = np.sqrt(gx**2 + gy**2)
+    Z = scalar/scalar.max()
```

Comment:
There is the raw version of the code lines featured in NOTES.

* As mentioned in issue #14 , use self-explanatory variable naming, hard-coded values should be passed as a variable, with a name that is also self-explanatory.
* Inline comments are missing too

Reply:
A separate function will come with the slider. To have a nice layout of the figures, it is considerably easier to do it this way. Should it be made optional in the future, it might be worth to have a draw superfunction that optionally calls draw_lagrangian_descriptor and optionally gradient. This would make it easier to the subplots.